internal/testutils/basefee/contracts/AuxBaseFee.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import "./Lib1559Math.sol";
+
+contract AuxBaseFee {
+    function baseFee(uint256 gasExcess_, uint8 basefeeAdjustmentQuotient, uint32 gasTargetPerL1Block) public pure returns (uint256 basefee_) {
```

Comment:
This function is aim to simulate [_calc1559BaseFee](https://github.com/taikoxyz/taiko-mono/blob/17abc189ca3d2752beb5400c036a650fd5b9c895/packages/protocol/contracts/L2/TaikoL2.sol#L252).